editor.planx.uk/package.json

```diff
@@ -81,6 +81,7 @@
     "react-navi-helmet-async": "^0.15.0",
     "react-toastify": "^9.1.3",
     "react-use": "^17.5.0",
+    "react-virtuoso": "^4.10.4",
```

Comment:
**Why Virtuoso?**
 - Recommended in [MUI docs](https://mui.com/material-ui/react-list/#virtualized-list)
 - I tried the more basic react-window first and quickly found myself having a lot of CSS and layout issues - Virtuoso "just worked"
 - Good score on [socket.dev](https://socket.dev/npm/package/react-virtuoso)
 - [Docs](https://virtuoso.dev/) were solid and clear